docs/platform/manager/dns-manager-add-domain-shortguide/index.md

```diff
@@ -25,10 +25,9 @@ Creating a domain also creates its corresponding domain zone.
 
     {{< image src="add-new-domain.png" alt="This page lets you add a new domain" title="This page lets you add a new domain" >}}
 
-1. If you want to add a *slave zone* instead of a master zone, click the **Slave** radio button.
+1. If you want to add a *slave zone* instead of a master zone, click the **Slave** radio button. If not, you may skip to the next step.
 
-    {{< note >}}
-In order for Linode's DNS servers to function as slaves, your DNS master server must notify and allow AXFR requests from the following IP addresses:
+In order for Linode's DNS servers to function as slaves,your DNS master server must notify and allow AXFR requests from the following IP addresses:
```

Comment:
space between `slaves,` and `your`

Reply:
Looks like this Note isn't rendering for some reason: https://deploy-preview-3690--nostalgic-ptolemy-b01ab8.netlify.app/platform/manager/dns-manager/#add-a-domain